client/src/types/typesDonu.ts

```diff
@@ -56,6 +57,9 @@ type ModelDefinition = {
   type: Type;
 }
 
+// TO-DO: Export GroMEt type from research/gromet/tools/types/GroMEt
```

Comment:
Could you precise the full URL, as I cannot find /research on the codebase.

Reply:
its not on the codebase, but it is part of the research repo

https://github.com/uncharted-aske/research/blob/main/gromet/tools/types/GroMEt.d.ts

Reply:
Thanks, now it will be accessible with Git blame.